tools/generate_og_images.py

```
@@ -0,0 +1,58 @@
+import os
+import sys
+import argparse
+from PIL import Image, ImageDraw, ImageFont
+
+def generate_og_image(content_type, output_dir):
```

Ruff (0.8.2)

Missing return type annotation for public function `generate_og_image`

Add return type annotation: `None`

(ANN201)